app/javascript/dashboard/components/widgets/forms/Input.vue

```diff
@@ -42,6 +43,10 @@ export default {
       type: String,
       default: '',
     },
+    readonly: {
+      type: String,
+      deafaut: '',
```

Comment:
Please change it to `default` and use a Boolean. `readonly` is a boolean attribute and please set the default value to false.

Reply:
@pranavrajs Oops missed that. Done.